rest_tools/client/client.py

```diff
@@ -71,18 +79,26 @@ def calculate_retries(self, timeout: float, backoff_factor: float) -> int:
             )
 
         # the first backoff is always 0 sec, factor applies after 2nd attempt
-        #    T + 0 + sum{n=1,retries-1}(T + 2^n * B)     + T
-        # =  T + 0 + ( T*k + B*2^((retries-1)+1) - 2*B ) + T
-        # not easily solvable (for k) in a closed form
-        for k in range(0, MAX_RETRIES + 2):  # last val -> MAX_RETRIES+1
-            if self.waittime_max > (
+        #    T  +  0  +  sum{n=1,retries-1}(T + min[MAX, 2^n * B] )  +  T
+        # sum has no closed form due to `min` function
+        for candidate in range(0, MAX_RETRIES + 2):  # last val is MAX_RETRIES+1
+            total = (
                 timeout
-                + (timeout * k)
-                + (backoff_factor * 2 ** (k))
-                - (2 * backoff_factor)
+                + 0
```

Comment:
Why is there a `+ 0`?  That just looks weird.

Reply:
it's there to remind myself that the first backoff is always 0